sw/device/lib/drivers/dma/dma.c

```diff
@@ -31,6 +31,11 @@
 /**                                                                        **/
 /****************************************************************************/
 
+#ifdef __cplusplus
```

Comment:
in this file only the ifdef part related to cpluspluc should have been changed, instead, it is full of spaces and tabs, enters and so on, can we pls make this back to the original plus the only ifdef changes?

Reply:
I think at some point the auto-formatter must have been activated, there is one more small change. I'll revert it back to the old format